README.md

```diff
@@ -1 +1,15 @@
-Insolar documentation sources
+# Insolar documentation sources
+
+## Install
+1. Install the latest Sphinx version: `python3 -m pip install sphinx`
+2. Set PATH if you installed it with the `--user` key: `PATH="/Users/<user>/Library/Python/<version>/bin:$PATH"`
+3. Install the extensions used in this project: 
+   ```
+   python3 -m pip install sphinx-tabs
```

Comment:
Make it a single command: python3 -m pip install sphinx-tabs sphinxcontrib.plantuml sphinx_copybutton sphinxcontrib.contentui

Reply:
done